src/include/common/rwlatch.h

```diff
@@ -80,6 +80,9 @@ class ReaderWriterLatch {
       }
     } else {
       if (reader_count_ == MAX_READERS - 1) {
+        // TODO: this implementation will lead to deadlock after reader count reaches
```

Comment:
```suggestion
        // TODO(Chi): this implementation will lead to deadlock after reader count reaches
```

We normally include someone in the TODO parens as, "if we need more information, this is the person who'd probably know"